### PR TITLE
Submit requests

### PR DESCRIPTION
**Pull request by maysakanoni**
**What?**

- Embedded the google-form

**Why?**

- So it is easier to navigate

- Can now  link back to the other view from the `submit_request` view

image

### CODE REVIEW

File: views/submit_request.erb [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    <header class="header">
+      <div class="header__inner">
+        <nav class="p-0 flex-column flex-lg-row navbar navbar-expand">
+          <div class="mr-lg-auto">
+            <div class="header__logo">
+              <a href="#" class="header__logo_link">
+                <div class="header_logo_type--by">
+                  <span class="header_logo_type__text">Requests</span>
+                  <span class="px-3">by</span>
+                  <span class="position-relative" style="top: -2px;">
```
Comment (georges1996) [inline views/submit_request.erb]: Would remove the inline style

File: views/submit_request.erb [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+                  <img alt="Made Tech" itemprop="logo" src="/assets/images/made-tech-logo-colour.png" width="120px">
```
Comment (georges1996) [inline views/submit_request.erb:34]: And here

File: app.rb [hunk 3]
```diff
@@ -28,3 +28,8 @@
   erb :resolved_requests, locals: { data: response }
 end
 
+get '/submit_request' do 
+  erb :submit_request
+end
+
+
```
Comment (georges1996) [inline app.rb:35]: Remove this extra line

**Review by georges1996 — CHANGES_REQUESTED** [review]

File: views/submit_request.erb (hunk 1 above)
Comment (maysakanoni) [reply views/submit_request.erb]: resolved in https://github.com/madetech/requestinator/commit/6211e1163bb0953f433e9a688b1a417fe181f785

File: views/submit_request.erb (hunk 2 above)
Comment (maysakanoni) [reply views/submit_request.erb:34]: resolved in https://github.com/madetech/requestinator/commit/6211e1163bb0953f433e9a688b1a417fe181f785

**Review by georges1996 — APPROVED** [review]

**Pull request merged**